packages/contracts/test/connect-contracts.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { ethers } from 'hardhat'
+import { Signer, Contract } from 'ethers'
+import {
+  connectL1Contracts,
+  connectL2Contracts,
+} from '../dist/connect-contracts.js'
+
+describe('connectL1Contracts', () => {
+  let user: Signer
+  before(async () => {
+    ;[user] = await ethers.getSigners()
+  })
+
+  it('should connect to all mainnet l1 contracts', async () => {
+    const l1Contracts = await connectL1Contracts(user, 'mainnet')
+    console.log(l1Contracts)
```

Comment:
I'm unable to get any further than this. I can't figure out what is causing this error:

`SyntaxError: Unexpected token x in JSON at position 1`

Reply:
Did you resolve this?

Reply:
yep Kelvin found the problem